src/Sylius/Bundle/ResourceBundle/EventListener/ORMTranslatableListener.php

```diff
@@ -214,4 +214,30 @@ private function hasUniqueConstraint(ClassMetadata $metadata, array $columns): b
 
         return false;
     }
+
+    /**
+     * @param object $translatableEntityLocaleAssigner
+     * @return TranslatableEntityLocaleAssignerInterface
+     */
+    private function processTranslatableEntityLocaleAssigner(object $translatableEntityLocaleAssigner): TranslatableEntityLocaleAssignerInterface
+    {
+        if ($translatableEntityLocaleAssigner instanceof ContainerInterface) {
+            @trigger_error(
+                sprintf('Passing an instance of "%s" is deprecated since 1.4. Use "%s" instead.',
```

Comment:
The error message is rather confusing...

Reply:
Because we don't explicitly tell the user why ?

Reply:
I think the main problem is that we ask the user to "use `TranslatableEntityLocaleAssignerInterface` instead of `ContainerInterface`" (???)